mm2src/coins/eth.rs

```diff
@@ -1119,6 +1119,20 @@ impl MarketCoinOps for EthCoin {
         )
     }
 
+    fn send_raw_tx_bytes(&self, mut tx: &[u8]) -> Box<dyn Future<Item = String, Error = String> + Send> {
+        if tx.starts_with(b"0x") {
```

Comment:
The `0x` prefix is used only to encode transaction in hex string representation. This check is not required here.